CONTRIBUTING.md

```diff
@@ -122,3 +122,28 @@ Also, add your new docs to `./docs/go`, say you add `introduction.mdx`
 Now, run `npm start` and you can access your Go docs at [http://localhost:3000/go/introduction](http://localhost:3000/go/introduction)
 
 Then, you can create versions and edit older versions as mentioned above.
+
+## Documentation Guidelines about Tools that are integrated with this documentation site
+
+### Lighthouse CI
+
+Lighthouse CI is a suite of tools that make continuously running, saving, retrieving, and asserting against [Lighthouse](https://github.com/GoogleChrome/lighthouse)
+results as easily as possible.
```

Comment:
I don't quite understand what this is saying -- Lighthouse is a suite of tools that do things with Lighthouse results.  What does it check for?